src/db/index.js

```diff
@@ -52,6 +54,39 @@ db.sync = () => {
   db.update('courses')
 }
 
+db.check = (callback) => {
```

Comment:
We'll probably tackle all callback patterns in one swoop sometime in the future. As of now, it's fine.